diff -up plasma-framework-5.84.0/src/declarativeimports/core/CMakeLists.txt.orig plasma-framework-5.84.0/src/declarativeimports/core/CMakeLists.txt
--- plasma-framework-5.84.0/src/declarativeimports/core/CMakeLists.txt.orig	2021-07-03 14:20:52.000000000 +0200
+++ plasma-framework-5.84.0/src/declarativeimports/core/CMakeLists.txt	2021-07-11 15:51:56.010833913 +0200
@@ -58,7 +58,7 @@ if(HAVE_X11)
 endif()
 
 if(HAVE_EGL)
-    target_link_libraries(corebindingsplugin ${EGL_TARGET})
+    target_link_libraries(corebindingsplugin GL EGL)
 else()
 
 endif()
